nengo/tests/test_learning_rules.py

```diff
@@ -138,8 +173,26 @@ def test_pes_neuron_ens(Simulator, plt, seed, rng, allclose):
         allclose,
         pre_neurons=True,
         post_neurons=False,
+        weight_solver=weight_solver,
```

Comment:
If `pre` is neurons, the solver shouldn't be used at all. Why test this?

Reply:
Because this test failed before these changes :) (because we were assuming, incorrectly, that if a weight solver is used then `to_neurons=True`)

Reply:
Got it. We should add a comment to that effect.

Reply:
Moved this case into the new `test_build_pes` (see below) instead.